ui/nuxeo-video/nuxeo-video-conversions.js

```diff
@@ -0,0 +1,84 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import '@polymer/iron-flex-layout/iron-flex-layout-classes.js';
+import { FormatBehavior } from '../nuxeo-format-behavior.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
+import '../nuxeo-icons.js';
+import '../widgets/nuxeo-tooltip.js';
+
+{
+  /**
+  `nuxeo-video-conversions`
+  @group Nuxeo UI
+  @element nuxeo-video-conversions
+  */
+  class VideoConversions extends mixinBehaviors([I18nBehavior, FormatBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style>
+          a,
+          a:active,
+          a:visited,
+          a:focus {
+            @apply --nuxeo-link;
+          }
+          a:hover {
+            @apply --nuxeo-link-hover;
+          }
+        </style>
+        <h3>[[label]]</h3>
+        <div>
+          <dom-repeat items="[[document.properties.vid:transcodedVideos]]" as="conversion">
+            <dom-if if="[[conversion.content]]">
+              <template>
+                <div class="layout horizontal center">
```

Comment:
IIRC in order to apply `layout horizontal center` and such classes that come from `import '@polymer/iron-flex-layout/iron-flex-layout-classes.js'` we should have `<style include="iron-flex">`.

Reply:
Anyway, I believe it is something we are not promoting anymore, correct @Gabez0r ?
Should we use mixins for this?

Reply:
Anyway, I believe it is something we are not promoting anymore, correct @Gabez0r ?
Should we use mixins for this?

Reply:
Indeed, we are missing an import and the include to have those classes working.

We've been gradually replacing the usage of iron flex layout classes (e.g. `class="flex"`) for the custom css mixins (e.g. `@apply --layout-flex`), so the second would be preferable for consistency purposes only (they both do the same).

Reply:
cool, done